specs/BUCK

```diff
@@ -745,7 +745,11 @@ osquery_gentable_cxx_library(
             "windows",
         ),
         (
-            "windows/pipes.table",
+            "windows/hvci_status.table",
+            "windows",
+        ),        
+        (
+            _SPECS_LOCATION + "/windows/pipes.table",
```

Comment:
I'm betting you _might_ need to remove this `_SPECS_LOCATION` as well.

Reply:
Removed!